docs/compute/git.md

```diff
@@ -0,0 +1,152 @@
+# Git integration 
+
+The Git integration allows users to manage and version control their feature engineering codebase and workflows using Git directly from the Hopsworks UI. The integration currently supports repositories hosted on Github, Gitlab and Bitbucket. 
+
+!!! warning "Beta"
+    The feature is currently in Beta and will be improved in the upcoming releases.
+
+## Credentials
+
+When you perform Git operations on Hopsworks that need to interact with the remote repository, Hopsworks relies on the Git HTTPs protocol to perform those operations. Authentication with the remote repository happens through a token generated by the Git repository hosting service (Github, Gitlab, Bitbucket).
+
+Documentation on how to generate a token for the supported Git hosting services is available here:
+
+- [Github](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+- [Gitlab](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
+- [Bitbucket](https://confluence.atlassian.com/bitbucketserver/http-access-tokens-939515499.html)
+
+Once the token has been generate, you need to provided it to Hopsworks so that it can be used to authenticate Git operations. In the _Account Settings_ page you can find the _Git Providers_ section. The Git provider section displays which providers have been already configured and can be used to clone new repositories.
```

Comment:
```suggestion
Once the token has been generated, you need to provide it to Hopsworks so that it can be used to authenticate Git operations. In the _Account Settings_ page you can find the _Git Providers_ section. The Git provider section displays which providers have been already configured and can be used to clone new repositories.
```